game-core/src/main/java/games/strategy/engine/framework/startup/ui/panels/main/MainPanel.java

```diff
@@ -69,7 +70,7 @@
       final Consumer<MainPanel> launchAction,
       final Supplier<Optional<ChatModel>> chatPanelSupplier,
       final Runnable cancelAction) {
-    this.chatPanelSupplier = chatPanelSupplier;
+    this.chatModelSupplier = chatPanelSupplier;
```

Comment:
You might want to rename the parameter as well

Reply:
It is not important relative to the problem fix, but okay. 02fe66387

The name change is important to the problem fix as the out of date variable name was used to infer the expected sub-class types.